measureme/src/raw_event.rs

```diff
@@ -1,48 +1,100 @@
 use crate::stringtable::StringId;
 
+/// `RawEvent` is how events are stored on-disk. If you change this struct,
```

Comment:
I would love to see a test or static assertion that verifies the size of `RawEvent` is what we think it is.

Reply:
That's a good idea!

Reply:
Done.